Makefile

```diff
@@ -39,13 +41,13 @@ gen:
 
 .PHONY: clean
 clean:
-	$(DOCKER_COMPOSE) down -v
+	make down
 	rm -rf $(PROJECT_ROOT)/webapp/src/shared/generated || true
 	rm $(PROJECT_ROOT)/transcendence-app/swagger-spec/swagger-spec.yaml || true
 
 .PHONY: prod-clean
 prod-clean:
-	$(DOCKER_COMPOSE) -f docker-compose.yml -f docker-compose.prod.yml down -v
+	make prod-down
```

Comment:
Not very sure if I am mistaken but, with this you'll finally execute:
`	$(DOCKER_COMPOSE) -f docker-compose.yml -f docker-compose.prod.yml down`
which will remove everything (including volumes where our DB data is stored)

So everytime we re-deploy our app all data will be deleted, right? (Despite of the knex migrations)